erpnext/accounts/doctype/account_category/account_category.py

```
@@ -21,6 +21,7 @@ class AccountCategory(Document):
 
 		account_category_name: DF.Data
 		description: DF.SmallText | None
+		root_type: DF.Literal["", "Asset", "Liability", "Income", "Expense", "Equity"]
```

Ruff (0.15.2)

[error] Syntax error in forward annotation: Expected an expression

(F722)